services/authentication/daos/roles_updates_dao.py

```diff
@@ -0,0 +1,26 @@
+from sqlalchemy import Column, String, Integer, DateTime, Boolean
+
+from db import Base
+
+class RolesUpdatesDAO(Base):
```

Comment:
You have a history table, very nice. Maybe rename it to RolesHistory? Makes it clear that this table has no effect on the service

Reply:
Agree, will change